From dae6d3e67f0cab5fff02c883e61dee797a79d759 Mon Sep 17 00:00:00 2001
From: Zhong Hongbo <hongbo.zhong@windriver.com>
Date: Fri, 1 Aug 2014 15:20:52 +0800
Subject: [PATCH] board/lsi: Add fdt_high and initrd_high variables

Using 0xffffffff implies that the initrd are initially copied
to working addresses. This will avoid an additional copy.
otherwise initrd will be relocated to random address

And change the console interface to /dev/ttyAMA0

Signed-off-by: Hongbo Zhong <hongbo.zhong@windriver.com>
---
 include/configs/axxia-55xx.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/include/configs/axxia-55xx.h b/include/configs/axxia-55xx.h
index 55944ff..7dbb6d3 100644
--- a/include/configs/axxia-55xx.h
+++ b/include/configs/axxia-55xx.h
@@ -135,7 +135,8 @@
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	"loadaddr=0x82000000\0" \
-	"console=ttyO2,115200n8\0" \
+	"console=/dev/ttyAMA0\0" \
+	"initrd_high 0xffffffff\0" \
 	"usbtty=cdc_acm\0" \
 	"vram=16M\0" \
 	"mmcdev=0\0" \
-- 
1.7.1

